link.go

```diff
@@ -28,6 +28,14 @@ type LinkAttrs struct {
 	Namespace    interface{} // nil | NsPid | NsFd
 }
 
+// NewLinkAttrs returns LinkAttrs structure filled with default values
+func NewLinkAttrs() LinkAttrs {
+	return LinkAttrs{
+		TxQLen: -1,
+		MTU:    -1,
+	}
+}
+
```

Comment:
Hmm, I'm not sure, but seems like we can't set `TxQLen` to `0` with netlink. `iproute2` uses `ioctl`.